### PR TITLE
add dyz60-hs

### PR DESCRIPTION
**Pull request by dayatz**
# dyz60_hs

60% layout with multiple layout and mounting support, it's similar to dyz60 but this is for hotswap version with different pin assignment
pcb

* Keyboard Maintainer: [dayatz](https://github.com/dayatz)
* Hardware Supported: dyz60, ATMega32u4
* Hardware Availability: Local Indonesia GB.

## Bootloader
Enter the bootloader in 3 ways:
* **Bootmagic reset**: Hold down the Esc key and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB (near spacebar key)
* **Physical reset pins**: Short the reset pins under spacebar key

## Flashing
Make example for this keyboard (after setting up your build environment):
    make dyz/dyz60_hs:default

Flashing example for this keyboard:
    make dyz/dyz60_hs:default:flash

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

### CODE REVIEW

File: keyboards/dyz/dyz60_hs/rules.mk [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+#
+BOOTMAGIC_ENABLE = yes     # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = no        # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = yes           # USB Nkey Rollover
```
Comment (drashna) [inline keyboards/dyz/dyz60_hs/rules.mk:18]: Please remove the `SLEEP_LED_ENABLE` setting, as it's not really used, and remove the comments for both it and NKRO.  
```suggestion
NKRO_ENABLE = no            # Enable N-Key Rollover
```


Also, most of the settings here can be moved to the info.json file.

File: keyboards/dyz/dyz60_hs/readme.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+![dyz60 pcb](https://i.imgur.com/l9z5KuJh.jpg)
+
+60% layout has flexcut with multiple layout and mounting support.
+
+* Keyboard Maintainer: [dayatz](https://github.com/dayatz)
+* Hardware Supported: dyz60, ATmega32U4
+* Hardware Availability: Local Indonesia GB.
+
+Make example for this keyboard (after setting up your build environment):
```
Comment (drashna) [inline keyboards/dyz/dyz60_hs/readme.md:11]: Could you add instructions on how to enter the bootloader, to flash new firmware?  This is needed, as a new user may not be aware of how to enter to bootloader to flash new firmware.

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```

**Review by drashna — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

File: keyboards/dyz/dyz60_hs/config.h [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+#define PRODUCT         dyz60-hs
+
+/* Debounce reduces chatter (unintended double-presses) - set 0 if debouncing is not needed */
+#define DEBOUNCE 5
+
+/* Bootmagic Lite key configuration */
+#define BOOTMAGIC_LITE_ROW 0
+#define BOOTMAGIC_LITE_COLUMN 0
+
+#define LED_CAPS_LOCK_PIN F1
```
Comment (zvecr) [inline keyboards/dyz/dyz60_hs/config.h:33]:
```suggestion
```

File: keyboards/dyz/dyz60_hs/info.json [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+        "led_count": 16,
+        "pin": "B7",
+        "hue_steps": 10,
+        "saturation_steps": 10,
+        "brightness_steps": 10,
+        "animations": {
+            "all": true
+        }
+    },
+    "usb": {
```
Comment (zvecr) [inline keyboards/dyz/dyz60_hs/info.json:20]:
```suggestion
    "indicators": {
        "caps_lock": "F1"
    },
    "usb": {
```

File: keyboards/dyz/dyz60_hs/config.h [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define MANUFACTURER    dyz
+#define PRODUCT         dyz60-hs
```
Comment (zvecr) [inline keyboards/dyz/dyz60_hs/config.h:24]:
```suggestion
```

File: keyboards/dyz/dyz60_hs/info.json [hunk 6]
```diff
@@ -0,0 +1,100 @@
+{
+    "keyboard_name": "dyz60-hs",
```
Comment (zvecr) [inline keyboards/dyz/dyz60_hs/info.json:2]:
```suggestion
    "keyboard_name": "dyz60-hs",
    "manufacturer": "dyz",
```

**Review by zvecr — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**